tests/test-cases/Group11-Upgrade/11-06-Upgrade-StorageQuota.robot

```diff
@@ -0,0 +1,341 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 11-06- Upgrade To latest Check storage quota feature
+Resource  ../../resources/Util.robot
+Suite Setup  Disable Ops User And Install VIC To Test Server
+Suite Teardown  Re-Enable Ops User And Clean Up VIC Appliance
+Default Tags
+
+*** Variables ***
+${run-as-ops-user}=  ${EMPTY}
+${old_version}=  v1.4.3
+
+*** Keywords ***
+Disable Ops User And Install VIC To Test Server
+    ${run-as-ops-user}=  Get Environment Variable  RUN_AS_OPS_USER  0
+    Set Environment Variable  RUN_AS_OPS_USER  0
+    Install VIC with version to Test Server  ${old_version}  additional-args=--container-name-convention VCH_1_{name} --cpu-reservation 1 --cpu-shares normal --memory-reservation 1 --memory-shares normal --endpoint-cpu 1 --endpoint-memory 2048 --base-image-size 8GB --bridge-network-range 172.16.0.0/12 --container-network-firewall vm-network:published --certificate-key-size 2048
+
+Re-Enable Ops User And Clean Up VIC Appliance
+    Set Environment Variable  RUN_AS_OPS_USER  ${run-as-ops-user}
+    Reload VCH Related Environment Variables  ${vchName1}  ${vchIP1}  ${vchParams1}  ${vchAdmin1}
+    Clean up VIC Appliance And Local Binary
+    Reload VCH Related Environment Variables  ${vchName2}  ${vchIP2}  ${vchParams2}  ${vchAdmin2}
+    Cleanup VIC Appliance On Test Server
+    Reload VCH Related Environment Variables  ${vchName3}  ${vchIP3}  ${vchParams3}  ${vchAdmin3}
+    Cleanup VIC Appliance On Test Server
+
+Create Containers For VCH_1
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name c-vm1 -id busybox
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --name c-vm2 -id nginx
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create --name c-vm3 -i nginx
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} volume create --opt Capacity=5G volume1
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+Custom Install VCH
+    [Arguments]  ${version}  ${prefix}  ${imageStore}=%{TEST_DATASTORE}  ${baseImageSize}=8G  ${cleanup}=${False}
```

Comment:
Please also make bridge network range as parameter and when you create 3 VCH, please assign different bridge network-range for them.